### ISSUE TITLE
Consuming messages from a simple consumer

### ISSUE THREAD

**Issue by reutshal** [Parsely/pykafka#968]
Hey,
I'm trying to consume messages from a topic using pykafka with no duplicates.
I'm interested in consuming messages every few minutes, so every time I will get only new messages.

 I tried this (as a part of a function that will be called every few minutes):
![image](https://user-images.githubusercontent.com/50294648/65182320-b8e5cd00-da69-11e9-898a-09733a4ed52e.png)

but I'm getting the same messages every time.

I also tried to set the consumer like this:
![image](https://user-images.githubusercontent.com/50294648/65182016-07df3280-da69-11e9-9bb0-60f52dcbf855.png)

with offset start from -2 and then update the offset according to the op.last_offset_consumed,
but it still didn't work.

Basically I tried to perform a very trivial Kafka pattern- consume new messages.

Any ideas for this? 
I will appreciate your quick response.

Thank you!
